QUARKUS-6262.MD

```diff
@@ -0,0 +1,61 @@
+# QUARKUS-6262 - Upgrade to Hibernate Validator 9.0
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6262
+
+Upstream issues:
+
+- https://github.com/quarkusio/quarkus/issues/5531
+- https://github.com/quarkusio/quarkus/issues/35144
+
+Upstream PR: https://github.com/quarkusio/quarkus/pull/42292
+
+PR #42292 extends Hibernate Validator support in Quarkus, covering two areas:
+1. Support for `validation.xml` constraints – although Quarkus does not support configuration of the `ValidatorFactory` via XML, constraint declarations in `validation.xml` must be respected.
+2. Method override rules – validation must ensure that a method overriding another cannot redefine the parameter constraint configuration (as required by Bean Validation specification).
+
+## Getting familiar with the features
```

Comment:
Usually, this section goes through the end of the TP "template" that we typically use